src/app/component/ProductCard/ProductCard.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+export const OPTION_TYPE_COLOR = '1';
```

Comment:
If you need something like this, prefer having 
```js
export const OPTION_TYPE_COLOR = 'OPTION_TYPE_COLOR';
```

Reply:
"type === '1' is not semantic enough.
Please define the constant with the meaningful name in the component config file and move type detection and appropriate component render to a separate render function."

From previous code review by Zhan